doc/guides/west/manifest.rst

```diff
@@ -1757,6 +1871,31 @@ If an imported manifest file has a ``west-commands:`` definition in its
 available extensions at the time the manifest is imported. They will thus take
 precedence over any extension commands with the same names added later on.
 
+Group filters
+-------------
+
+The resolved manifest has a ``group-filter`` value which is the result of
+concatenating the ``group-filter`` values in the top-level manifest and any
+imported manifests.
+
+Manifest files which appear earlier in the import order have higher precedence
+and are therefore concatenated later into the final ``group-filter``.
+
+In other words, let:
+
+- the submanifest resolved from ``self-import`` have group filter ``self-filter``
+- the top-level manifest file have group filter ``top-filter``
+- the submanifests resolved from ``import-1`` through ``import-N`` have group
+  filters ``filter-1`` through ``filter-N`` respectively
+
+The final resolved ``group-filter`` value is then ``filter1 + filter-2 + ... +
+filter-N + top-filter + self-filter``, where ``+`` here refers to list
+concatenation.
```

Comment:
I am missing something here. From the example in the "Group Filters and Imports" section, the sequence would be:
`group-filter = [-unstable] +  [+unstable,-optional]`, which leads to `[+unstable, -optional]`.

But if the order would have been:
`group-filter = [+unstable] +  [-unstable,-optional]` then the end result would have had to be `[-unstable,-optional]`.
Is this captured in the resolution sequence of additions above?

Reply:
i.e. this is not a standard commutative sum, which I am not sure is well reflected.

Reply:
> i.e. this is not a standard commutative sum, which I am not sure is well reflected.

I've tried to address this.